polar-core/src/error.rs

```diff
@@ -389,18 +389,22 @@ impl fmt::Display for RuntimeError {
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub enum OperationalError {
-    Unimplemented(String),
+    Unimplemented {
```

Comment:
This could change the serialization format of these errors (I think). Does that mean we need to change deserialization code in how libraries?